illumidesk/index.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
I'm probably missing something but am unsure of the purpose of this index.yaml. It looks to me like it should just be a vanilla reference to the JupyterHub helm chart. Could you confirm?

Reply:
Accidentally included index.yaml when creating the helm chart repo. Removed